models/userStatus.js

```diff
@@ -371,7 +372,7 @@ const batchUpdateUsersStatus = async (users) => {
     const statusToUpdate = {
       state,
       message: "",
-      from: currentTimeStamp,
+      from: new Date().setUTCHours(0, 0, 0, 0),
```

Comment:
Generally if we mark someone as OOO or IDLE or ACTIVE its a good idea to keep the start of the day as the time rather than any random timestamp of the day as we do in progresses and other collection.